public/app/features/explore/Logs/LogsNavigation.tsx

```diff
@@ -7,6 +7,7 @@ import { config, reportInteraction } from '@grafana/runtime';
 import { DataQuery, TimeZone } from '@grafana/schema';
```

Comment:
Main change here: added scrollToBottomLogs prop, that when it's present shows a button to scroll to the bottom. Not provided for the current old panel.